game/scripts/npc/items/item_rapier.txt

```diff
@@ -13,7 +13,7 @@
 
 		// Item Info
 		//-------------------------------------------------------------------------------------------------------------
-		"ItemCost"						"50000"
+		"ItemCost"						"150000"
```

Comment:
Internally, the custom gold system still uses normal Dota gold for things like buying from shop. Fixing this would require making the shop use custom gold directly. Depending on how the Black Star shop was actually done, this may be easy or could take a bit of work.